stdlib/3/tempfile.pyi

```diff
@@ -164,6 +164,39 @@ else:
             dir: Optional[_DirT[AnyStr]] = ...,
         ) -> IO[Any]: ...
 
+class _TemporaryFileWrapper(IO[str]):
+    file: IO[str]
+    name: Any
+    delete: bool
+    def __init__(self, file: IO[str], name: Any, delete: bool = ...) -> None: ...
+    def __enter__(self) -> _TemporaryFileWrapper: ...
+    def __exit__(
+        self, exc: Optional[Type[BaseException]], value: Optional[BaseException], tb: Optional[TracebackType]
+    ) -> Optional[bool]: ...
+    def __getattr__(self, name: str) -> Any: ...
+    def close(self) -> None: ...
+    def unlink(self, path: str) -> None: ...
```

Comment:
Where does this come from? As far as I can see, it's not defined in `_TemporaryFileWrapper`, so shouldn't it be below the next comment?